apps/storybook/src/ProfileSkills.stories.tsx

```diff
@@ -8,6 +8,10 @@ export default {
 
 export const Normal = () => (
   <ProfileSkills
+    skillsDescription={text(
+      'Description',
+      'Multiple years of experience in Parkinson.',
```

Comment:
This sounds like experience from having the disease :see_no_evil: 
```suggestion
      "Multiple years of experience in research on Parkinson's Disease.",
```